src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/WhitelistApiTest.java

```diff
@@ -21,6 +21,7 @@
 import com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditMessage;
 import com.amazon.opendistroforelasticsearch.security.auditlog.integration.TestAuditlogImpl;
 import com.amazon.opendistroforelasticsearch.security.dlic.rest.validation.AbstractConfigurationValidator;
+import com.amazon.opendistroforelasticsearch.security.filter.OpenSearchSecurityRestFilter;
```

Comment:
The `@link com.amazon.opendistroforelasticsearch.security.filter.OpenDistroSecurityRestFilter` is currently not working without importing the class.